### ISSUE TITLE
Doesn't block entered URLs

### ISSUE THREAD

**Issue by erasercat** [realityfabric/Tumblr-Batch-Block#12]
As of a few days ago the extension stopped working completely, when entering URLs to block nothing happens. I tried reinstalling, refreshing and re-entering about 10 URLs but they still wouldn't automatically block. 😣

**Comment by realityfabric** [comment]
Do you have a lot of users blocked already? What happens when you try to block?

The block list should open and load all the users you've blocked, does that happen?

**Comment by erasercat** [comment]
Yes, I have around... 10-20,000 users blocked? :") I can enter URLs myself and they block just fine but not when I put them into the batch block.

Putting the list into batch block opens the block list but after that nothing appears to happen after that.

**Comment by realityfabric** [comment]
Ah, then this is probably a duplicate of #11.

I changed the way blocking works, and didn't consider how it might impact users with way more blogs blocked than me.

Until I'm able to fix the problem, try installing an older version. You can find a link at #11, let me know if you need any help.

**Comment by erasercat** [comment]
Oh of course! Sorry for opening a new issue I thought it was different ,, thank you for your patience and help! :"D
